frontend-project/src/pages/letters/LettersDetailView.tsx

```diff
@@ -1,14 +1,17 @@
+import { Uploader } from '@/components/Uploader/Uploader';
+import { FileService } from '@/services/files';
+import { PostData } from '@/services/service';
 import { PageHeaderWrapper } from '@ant-design/pro-layout';
-import { Button, Card, Col, DatePicker, Form, Input, Row, Select, Spin, Switch } from 'antd';
+import { Button, Card, Col, DatePicker, Form, Input, Row, Select, Space, Spin, Switch } from 'antd';
 import moment from 'moment';
-import React, { useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { formatMessage, FormattedMessage } from 'umi-plugin-react/locale';
 import router from 'umi/router';
 import { FetchSelect } from '../../components/FetchSelect';
 import { localeKeys } from '../../locales/pl-PL';
 import { DetailMatchParam } from '../../models/connect';
 import { AutocompleteService } from '../../services/autocomplete';
-import { Letter } from '../../services/definitions';
+import { File as ResourceFile, Letter } from '../../services/definitions';
```

Comment:
Dałoby się nazwę tego interfejsu poprawić u źródła? Tj. widzę, że `This file is generated, do not manually edit this file` (skąd się właściwie biorą te definicje, z czego one są generated?).

Taka nazwa interfejsu to proszenie się o kłopoty.

Reply:
Generowane na postawie pliku OpenAPI (swagger.json) :(
Trzeba by zmienić generator pod to.